Source/Bogus.Tests/DataSetTests/DateTest.cs

```diff
@@ -10,7 +10,7 @@ public class DateTest : SeededTest
    {
       public DateTest()
       {
-         date = new Date();
```

Comment:
I'd avoid whitespace changes in PR's. You can actually set git to ignore them if you prefer: see core.whitespace (https://git-scm.com/book/en/v2/Customizing-Git-Git-Configuration)